fatal/type/test/reverse_test.cpp

```diff
@@ -0,0 +1,158 @@
+/*
+ *  Copyright (c) 2016, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#include <fatal/type/list.h>
+#include <fatal/type/pair.h>
+#include <fatal/type/reverse.h>
+
+#include <fatal/test/driver.h>
+
+namespace fatal {
+
+template <int> struct t {};
+using t0 = t<0>; using t1 = t<1>; using t2 = t<2>; using t3 = t<3>; using t4 = t<4>;
+using t5 = t<5>; using t6 = t<6>; using t7 = t<7>; using t8 = t<8>; using t9 = t<9>;
+template <int> struct u {};
+using u0 = u<0>; using u1 = u<1>; using u2 = u<2>; using u3 = u<3>; using u4 = u<4>;
+using u5 = u<5>; using u6 = u<6>; using u7 = u<7>; using u8 = u<8>; using u9 = u<9>;
+
+FATAL_TEST(reverse, list) {
```

Comment:
Change this to `reverse, pair`? Internal test runner fails if two tests have the same name.